parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -57,6 +57,16 @@ write_files:
   {{end}}
 {{end}}
 
+{{- if .MasterProfile.IsUbuntu2004}}
```

Comment:
Is this because of the `netplan apply` foo in CSE that we do for 20.04?

Reply:
No, it is because of the same reason 18.04 needs it.

My goal was to be as compatible to 18.04 as possible while getting to 20.04

The `netplan apply` foo is due to some annoying behavior of the cloud-init and the ip address assignment process.